google/cloud/bigquery/dbapi/cursor.py

```diff
@@ -212,6 +262,33 @@ def _try_fetch(self, size=None):
 
         if self._query_data is None:
             client = self.connection._client
+            bqstorage_client = self.connection._bqstorage_client
+
+            if bqstorage_client:
+
+                # TODO: Are we supposed to read somehting here in order to
```

Comment:
I recall that the error with small results happens at `create_read_session` time, so I don't think reading anything is necessary.

Reply:
Indeed, that error already already happens before streaming the rows, that comment was already dated. :)